src/Microsoft.ML.Api/StatefulFilterTransform.cs

```diff
@@ -98,7 +98,9 @@ private StatefulFilterTransform(IHostEnvironment env, StatefulFilterTransform<TS
 
         public bool CanShuffle { get { return false; } }
 
-        public Schema Schema => _bindings.Schema;
+        Schema IDataView.Schema => OutputSchema;
```

Comment:
>OutputSchema [](start = 35, length = 12)

I would still point on _binding.Schema, rather than OutputSchema.

Reply:
What's the difference? @tfinley@gmail.com, any comment?

---
In reply to: [237949493](https://github.com/dotnet/machinelearning/pull/1759#discussion_r237949493) [](ancestors = 237949493)

Reply:
You have one less hop between data access.
And in generally it creates that weird dependency where IDataView.Schema=>OutputSchema=>_binding.Schema and if you decide to change definition of OutputSchema, and it will part away from definition of IDataView.Schema, it would be really easy to make mistake.

---
In reply to: [237973483](https://github.com/dotnet/machinelearning/pull/1759#discussion_r237973483) [](ancestors = 237973483,237949493)

Reply:
I would personally honestly have written it the way @wschin did, just to emphasize the equivalence between them. I have to imagine for a release build it won't really make a difference.

---
In reply to: [237996719](https://github.com/dotnet/machinelearning/pull/1759#discussion_r237996719) [](ancestors = 237996719,237973483,237949493)